docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,84 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since libsodium's tests do not provide reference data for the SealedBox
+construction, the implementation is verified with a ``sealbox_test_vectors``
+utility program that produces and checks custom test vectors
+by making specific calls to libsodium API.
+
+To build the ``sealbox_test_vectors`` you need a ``C`` language compiler,
+a prebuilt libsodium library more recent than version 1.0.3 and the
+corresponding include headers.
+
+In a unix-like programming environment you shoud then execute:
+
+.. code-block:: bash
+
+    $ cc -o sealbox_test_vectors -l sodium sealbox_test_vectors.c
+
+The utility program can both generate list of vectors on standard
+output and check consistency of vectors pulled from standard input.
+
+Vector generation
+"""""""""""""""""
+
+When called with one or more command line arguments, ``sealbox_test_vectors``
+will generate the number of hex-encoded vectors requested as first argument,
+with the optional second and third arguments influencing the length of the
+generated messages:
+
+.. code-block:: bash
+
+    $ ./sealbox_test_vectors 1
+    XXXX...        XXXX...        XXXX...        XXXX...
+
+The second argument, if present, sets both a minimum and a maximum length
+on generated messages overriding the default 128 bytes values
+respectively with the supplied value and with twice the supplied value.
+
+The third argument, if present, sets the maximum length of generated
+messages.
+
+Vector test
+"""""""""""
+
+When called without command line arguments, ``sealbox_test_vectors``
+will parse and hex-decode the lines given as standard input and
+check if decoding the encrypted message will return the original
+message. A "OK"/"KO" tag will be appended to the input line to
```

Comment:
For failed test, I think `FAIL` or `ERROR` is more descriptive message than `KO` as someone might assume it's a typo for `OK`. Alternatively, if someone ends up parsing lots of results, It's hard to say how many errors occurred; 

    OK OK OK OK OK KO OK OK KO OK OK 
    
vs 

    OK OK OK OK OK FAIL OK OK FAIL OK OK

Reply:
Now I write ``FAIL`` on failure, as you suggested.